<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowWidget</class>
 <widget class="QWidget" name="WindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimulationWidget</string>
  </property>
  <widget class="SimulationWidget" name="simulationWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="paused">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PAUSED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="state">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial Ball State:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_single">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>single ball</string>
   </property>
  </widget>
  <widget class="QLabel" name="position">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="velocity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>velocity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="acceleration">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>acceleration:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="acc_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="acc_y">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="acc_z">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vel_y">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vel_z">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vel_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pos_y">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pos_z">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pos_x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="paused_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>W A S D - horizontal movement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paused_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>E Q - vertical movement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paused_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SPACE - default reset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="disclaimer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(only in single ball scene)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paused_6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC - pause</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_double">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>multi ball</string>
   </property>
  </widget>
  <widget class="QLabel" name="paused_7">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Mouse - orientation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SimulationWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>simulationwidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
